.ci/pull-request-config.yaml

```
@@ -8,6 +8,7 @@ pre: |
 
 default:
   build-command:
+    current: mvn -e clean install ${{ env.BUILD_MVN_OPTS }}
     upstream: mvn -e clean install -DskipTests -Dquickly ${{ env.BUILD_MVN_OPTS }}
```

skipTests not needed normally with `quickly` profile

```suggestion
    upstream: mvn -e clean install -Dquickly ${{ env.BUILD_MVN_OPTS }}
```